deployer/README.md

```diff
@@ -852,15 +852,15 @@ and existing deployments when changing the cluster size.
 Changing cluster parameters (or any parameters/secrets, really) requires
 re-deploying the instances. In order to minimize resynchronization
 between the instances as they are restarted, we advise halting traffic to
-ElasticSearch and then taking down the entire cluster for maintenance. No
+Elasticsearch and then taking down the entire cluster for maintenance. No
 logs will be lost; Fluentd simply blocks until the cluster returns.
 
-Halting traffic to ElasticSearch requires scaling down Kibana and removing node labels for Fluentd:
+Halting traffic to Elasticsearch requires scaling down Kibana and removing node labels for Fluentd:
 
     $ oc label node --all logging-infra-
     $ oc scale rc/logging-kibana-1 --replicas=0
```

Comment:
Scaling the RCs is not going to work anymore - you should direct all actions on DCs.